tests/test_user_models.py

```diff
@@ -296,8 +296,8 @@ def test_fail_get_unknown_service():
 @pytest.mark.parametrize("number,extra_service_code", (
     (1, "AR"),
     (2, "MP"),
-    (19, "VD"),
```

Comment:
acho uma boa ter um teste que tenta criar/pegar um serviço com código antigo (19) e falha

Reply:
Already have one that gets an invalid code: https://github.com/olist/correios/blob/fe3520bc451a672e4d9c812d514f39e13fe9d231/tests/test_user_models.py#L291

Reply:
boa, então sugiro adicionar mais um caso desse teste que vá além do 0 e teste também o 19, o objetivo desse teste é "impedir" que futuramente o 19 volte a ser um código válido. da maneira que os testes estão agora o 19 deixou de ser válido, mas não é testado isso, apenas que o 62 é válido

Reply:
Nice one!